src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,86 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_ITEMS_QUANTITY = 10;
+    private Pair elements;
+    private int size = 0;
+    private Pair[] storage;
+
+    public StorageImpl() {
+        storage = new Pair[MAX_ITEMS_QUANTITY];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAX_ITEMS_QUANTITY; i++) {
+            elements = new Pair(key, value);
+            if (isNull(storage[i])) {
+                storage[i] = elements;
+                size++;
+                break;
+            } else if (isKeyNull((storage[i])) && !isKeyNull((elements))) {
+                continue;
+            } else if (areKeysEqual(storage[i], elements)) {
+                storage[i] = elements;
+                break;
+            }
+        }
     }
 
     @Override
     public V get(K key) {
+        for (Pair pair : storage) {
```

Comment:
Use one `if` inside the loop
```suggestion
        for (int i = 0; i < size; i++) {
```

Reply:
+